deployments/bridges/poa-rialto/docker-compose.yml

```diff
@@ -79,12 +81,14 @@ services:
   # Note: These are being overridden from the top level `monitoring` compose file.
   prometheus-metrics:
     volumes:
-      - ./bridges/poa-rialto/dashboard/prometheus/:/etc/prometheus/
+      - ./bridges/poa-rialto/dashboard/prometheus/prometheus.yml:/etc/prometheus/prometheus-poa-rialto.yml
     depends_on: *all-nodes
 
   grafana-dashboard:
     volumes:
-       - ./bridges/poa-rialto/dashboard/grafana/provisioning/:/etc/grafana/provisioning/
+       - ./bridges/poa-rialto/dashboard/grafana/relay-poa-to-rialto-exchange-dashboard.json:/etc/grafana/provisioning/dashboards/relay-poa-to-rialto-exchange-dashboard.json
+       - ./bridges/poa-rialto/dashboard/grafana/relay-poa-to-rialto-headers-dashboard.json:/etc/grafana/provisioning/dashboards/relay-poa-to-rialto-headers-dashboard.json
+       - ./bridges/poa-rialto/dashboard/grafana/relay-rialto-to-poa-headers-dashboard.json:/etc/grafana/provisioning/dashboards/relay-rialto-to-poa-headers-dashboard.json
```

Comment:
Yes. When you run multiple bridges, the folder is populated by several compose files

Reply:
AFAIU the `volumes` definitions get's merged, so that we need multiple sources, but listing all the files here is sub-optimal. Can we have a (sub-) folder here instead?